common/membership/ringpop/factory.go

```diff
@@ -52,53 +53,44 @@ const (
 	persistenceOperationTimeout = 10 * time.Second
 )
 
-// factory provides a Monitor
+type factoryParams struct {
+	fx.In
+
+	Config          *config.Membership
+	ServiceName     primitives.ServiceName
+	ServicePortMap  config.ServicePortMap
+	Logger          log.Logger
+	MetadataManager persistence.ClusterMetadataManager
+	RPCConfig       *config.RPC
+	TLSFactory      encryption.TLSConfigProvider
+	DCCollection    *dynamicconfig.Collection
```

Comment:
If we're trying to establish a pattern to spread to other parts of the codebase, I'd want to call dymamicconfig.Collection just `DC`. `Collection` is adding very little value to the name (there's Client and Collection but Collection is the primary interface)

Reply:
I agree. I'll change it in the next push